v-spec.adoc

```diff
@@ -3175,8 +3175,9 @@ sequence `vmv.v.i vd, 0; vmerge.vim vd, vd, 1, v0`.
 NOTE: The vector integer move instructions share the encoding with the vector
 merge instructions, but with `vm=1` and `vs2=v0`.
 
-NOTE: The form `vmv.v.v vd, vd`, which does not change architectural
-state, is used as a HINT to indicate that the register will be used
+NOTE: The form `vmv.v.v vd, vd`, which does not change the state of
+non-tail elements of `vd`,
```

Comment:
I agree with HINT -> hint.

An attempt to word with fewer negatives given restart elements by definition are left unchanged:
>which leaves body elements in `vd`unchanged 

perhaps, given the only specified registers in vmv are vd and vs1, and for this case are equal, this should  even be reduced to
>which leaves body elements unchanged 

The rest of the explanation makes this formulation unambiguous.